.github/workflows/IDC_1100_Public_CI.yml

```diff
@@ -79,13 +80,14 @@ jobs:
         run: |
```

Comment:
line:74 https://github.com/intel/intel-extension-for-openxla/pull/31/files#diff-521cbe66e58dac318e2db11932867760ceedfa3f1f942c8ce604de148ca6ae24R74 should be changed to `pip install --force-reinstall *.whl` to make sure it's really reinstalled.